gnd/src/main/java/com/google/android/gnd/ui/home/featuresheet/EventHandler.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.ui.home.featuresheet;
+
+import com.google.android.gnd.model.observation.Record;
+
+public interface EventHandler {
```

Comment:
I think we can replace this interface `Consumer<Record>`. This way we also don't need to create a new interface for other events.